roles/common/tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     - empty.yml
   tags:
     - vars
+    # We need these vars for the entitlements tag to work
+    - entitlements
```

Comment:
Use the tag 'always'.  This is a magic ansible tag that ensures this will run whenever --tags is used.  I've done this in other places where vars are loaded as well.

If you do that, the 'vars' tag is unneeded.